lib/driver-helper.js

```diff
@@ -45,6 +45,38 @@ export function clickWhenClickable( driver, selector, waitOverride ) {
 	);
 }
 
+export function waitTillFocused( driver, selector, pollingOverride, waitOverride ) {
```

Comment:
@bsessions85 and @Stojdza so far, I addressed 2 out of 5 sleep statements in this PR. I used `waitTillFocused` method that I wrote. I remove the `click` from it (originally had it) since it is not needed. 
I placed the remaining 3 sleep statements that I originally removed back for now.
I think it is best if I address the remaining 3 in this file in a separate PR since the existing one already has a long history and it is getting hard to follow.